### ISSUE TITLE
"Passkey" for Login

### ISSUE THREAD

**Issue by HeinzWuert** [kimai/kimai#4138]
### Describe the problem

I would like to get the Passkey login process for Kimai.
There are a lot of ways to securely log into a system.
The simplest and most user-friendly variant to date is passkey.
Passkey is a variant of Fido, extremely easy to use and at the same time has a very high level of security.

### Describe the solution you'd like

https://www.heise.de/select/ct/2023/13/2312508413860013019

**Comment by MartinRyx** [comment]
I totally agree. This would be a great feature. Perfect when I need to login on some other than my own computer where I don't have my password manager. Entering a 30 chars random password isn't very much fun
